fire_forecast/data_preparation/select_data_all_shifts.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+from pathlib import Path
+
+import xarray as xr
+
+from fire_forecast.data_preparation.data_selector import DataSelector
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description="Script to select data from dataset.")
+    parser.add_argument("era5_path", type=str, help="Path to ERA5 dataset.")
+    parser.add_argument("era5_sfc_path", type=str, help="Path to fire dataset.")
+    parser.add_argument("gfas_path", type=str, help="Path to output dataset.")
+    parser.add_argument("output_path", type=str, help="Path to output dataset.")
+    parser.add_argument(
+        "--latitude_window_size", type=int, default=3, help="Latitude window size."
+    )
+    parser.add_argument(
+        "--longitude_window_size", type=int, default=3, help="Longitude window size."
+    )
+    parser.add_argument("--fire_threshold", type=int, default=1, help="Fire threshold.")
+    parser.add_argument(
```

Comment:
same as above:
can be misunderstood as value of areafraction which should be the threshold. Better rename e.g. 'threshold amount measurementdays'. Or similar